ports/esp32/boards/sdkconfig.ble

```diff
@@ -9,9 +9,15 @@ CONFIG_BT_NIMBLE_ENABLED=y
 CONFIG_BT_NIMBLE_MAX_CONNECTIONS=4
 
 # Pin to the same core as MP.
-# Until we move to IDF 4.2+, we need NimBLE on core 0, and for synchronisation
-# with the ringbuffer and scheduler MP needs to be on the same core.
-# See https://github.com/micropython/micropython/issues/5489
+# Prior to IDF 4.2+, we needed to pin NimBLE to core 0 for 
+# synchronization with the ringbuffer. This 
+#if ESP_IDF_VERSION >= ESP_IDF_VERSION_VAL(4, 2, 0)
```

Comment:
I'm pretty sure #if's are not supported in Kconfig files... I guess the only way to deal with this is to do the selection in the CMakeLists.txt file, which is getting tricky.

Reply:
I wondered.  I'll just pin to 1 then. :bowing_man:

Reply:
But now it won't work with IDF < 4.2.

Reply:
What's the path forward then?  Does my original intent to pin opposite of WiFi make things _worse_ for BLE and we just need to wait for that other thing....or what do you think we should do now?

Reply:
Two options:
1. wait for #7046 (it's very close) and then keep NimBLE on core 0
2. make CMakeLists.txt include the right sdkconfig fragment to pin NimBLE depending on the IDF version

Option (2) is not that difficult, but might anyway become obsolete once option (1) is in.

Reply:
Ok, with that in mind, I can put NimBLE back on core0 and we can wait this one out.  :)  

Thanks again for the reviews and help and advice here.